schemas/constructs/v1beta1/component.json

```diff
@@ -89,28 +160,171 @@
         },
-        "svgColor": {
+        "border-opacity": {
+          "type": "number",
+          "description": "The opacity of the node\u2019s border",
+          "minimum": 0,
+          "maximum": 1
+        },
+        "padding": {
+          "type": "number",
+          "description": "The amount of padding around all sides of the node.",
+          "minimum": 0
+        },
+
+        "opacity": {
+          "type": "number",
+          "description": "The opacity of the element. See https://js.cytoscape.org/#style/visibility",
+          "minimum": 0,
+          "maximum": 1
+        },
+        "z-index": {
+          "type": "integer",
+          "description": "An integer value that affects the relative draw order of elements. In general, an element with a higher z-index will be drawn on top of an element with a lower z-index. Note that edges are under nodes despite z-index."
+        },
+        "label": {
           "type": "string",
-          "description": "Colored SVG of the component used for UI representation on light background."
+          "description": "The text to display for an element\u2019s label. Can give a path, e.g. data(id) will label with the elements id"
         },
-        "svgComplete": {
+
+        "color": {
           "type": "string",
-          "description": "Complete SVG of the component used for UI representation, often inclusive of background."
+          "description": "The color of the element's label. Colours may be specified by name (e.g. red), hex (e.g. #ff0000 or #f00), RGB (e.g. rgb(255, 0, 0)), or HSL (e.g. hsl(0, 100%, 50%))."
+        },
+        "text-opacity": {
+          "type": "number",
+          "description": "The opacity of the label text, including its outline.",
+          "minimum": 0,
+          "maximum": 1
+        },
+        "font-family": {
+          "type": "string",
+          "description": "A comma-separated list of font names to use on the label text."
+        },
+        "font-size": {
+          "type": "string",
+          "description": "The size of the label text."
+        },
+        "font-style": {
+          "type": "string",
+          "description": "A CSS font style to be applied to the label text."
+        },
+        "font-weight": {
+          "type": "string",
+          "description": "A CSS font weight to be applied to the label text."
+        },
+        "text-transform": {
+          "type": "string",
+          "description": "A transformation to apply to the label text",
+          "enum": ["none", "uppercase", "lowercase"]
+        },
+        "text-halign": {
+          "type": "string",
+          "description": "The horizontal alignment of a node\u2019s label",
+          "enum": ["left", "center", "right"]
+        },
+        "text-valign": {
+          "type": "string",
+          "description": "The vertical alignment of a node\u2019s label",
+          "enum": ["top", "center", "bottom"]
+        },
+        "ghost": {
```

Comment:
@aabidsofi19, can you offer justification as to why this should be an explicit property and not an interpretation of other properties, like `isAnnotation`?

Reply:
@leecalcote  this is not ghost node property , but ghost is a style property that add shadow effect